tyr/autoscaler/autoscaler.py

```diff
@@ -0,0 +1,115 @@
+from boto.ec2.autoscale import AutoScaleConnection
+from boto.ec2.autoscale import LaunchConfiguration
+from boto.ec2.autoscale import AutoScalingGroup
+import boto.ec2
+import logging
+
+
+class AutoScaler(object):
+    '''
+    Autoscaler setup class
+    '''
+
+    log = logging.getLogger('Clusters.AutoScaler')
```

Comment:
This module should either be moved to the `tyr.clusters` package or the logger name should be changed to correctly reflect the package it belongs to - perhaps `Autoscaling.Autoscaler` (if the package were renamed `autoscaling`).